pkg/commands/oscommands/os_default_platform.go

```diff
@@ -5,14 +5,21 @@ package oscommands
 
 import (
 	"runtime"
+
+	"github.com/jesseduffield/lazygit/pkg/config"
 )
 
-func GetPlatform() *Platform {
-	return &Platform{
+func GetPlatform(osConfig config.OSConfig) *Platform {
+	platform := Platform{
 		OS:              runtime.GOOS,
 		Shell:           "bash",
 		ShellArg:        "-c",
 		OpenCommand:     "open {{filename}}",
 		OpenLinkCommand: "open {{link}}",
 	}
+	if osConfig.Shell != "" {
+		platform.Shell = osConfig.Shell
+		platform.ShellArg = osConfig.ShellArg
+	}
```

Comment:
I think at this point, we should probably just consider setting `bash` and `-c` as default parameters for `Shell` and `ShellArg` in the `user_config` file. That way, we can just directly take the values from `osConfig` and use them in the `Platform` literal.